docs/conf.py

```diff
@@ -1,274 +1,66 @@
 
-# Add any Sphinx extension module names here, as strings. They can be
-# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
-# ones.
 extensions = [
     "sphinx.ext.autodoc",
+    "sphinxcontrib.spelling",
+    "sphinx.ext.intersphinx",
+    "sphinx_copybutton",
 ]
 
-if not on_rtd:
-    extensions.append("sphinxcontrib.spelling")
+intersphinx_mapping = {
+    "python": ("https://docs.python.org/3/", None),
+}
 
-# Add any paths that contain templates here, relative to this directory.
 templates_path = ["_templates"]
+exclude_patterns = ["_build", ".DS_Store"]
+
+html_theme = "sphinx_wagtail_theme"
+html_theme_path = [sphinx_wagtail_theme.get_html_theme_path()]
+html_theme_options = {
+    "project_name": "Willow",
+    # "logo": "",
+    "github_url": "https://github.com/wagtail/Willow/tree/main/docs",
+    "footer_links": "",
+}
+html_last_updated_fmt = "%b %d, %Y"
 
-# The suffix of source filenames.
-source_suffix = ".rst"
-
-# The encoding of source files.
-# source_encoding = 'utf-8-sig'
-
-# The master toctree document.
-master_doc = "index"
-
-# General information about the project.
-project = "Willow"
-copyright = "2017, Torchbox"
-
-# The version info for the project you're documenting, acts as replacement for
-# |version| and |release|, also used in various other places throughout the
-# built documents.
-#
-# The short X.Y version.
-version = __version__
-# The full version, including alpha/beta/rc tags.
-release = __version__
-
-# The language for content autogenerated by Sphinx. Refer to documentation
-# for a list of supported languages.
-# language = None
-
-# There are two options for replacing |today|: either, you set today to some
-# non-false value, then it is used:
-# today = ''
-# Else, today_fmt is used as the format for a strftime call.
-# today_fmt = '%B %d, %Y'
-
-# List of patterns, relative to source directory, that match files and
-# directories to ignore when looking for source files.
-exclude_patterns = ["_build"]
-
-# The reST default role (used for this markup: `text`) to use for all
-# documents.
-# default_role = None
-
-# If true, '()' will be appended to :func: etc. cross-reference text.
-# add_function_parentheses = True
-
-# If true, the current module name will be prepended to all description
-# unit titles (such as .. function::).
-# add_module_names = True
-
-# If true, sectionauthor and moduleauthor directives will be shown in the
-# output. They are ignored by default.
-# show_authors = False
-
-# The name of the Pygments (syntax highlighting) style to use.
-pygments_style = "sphinx"
-
-# A list of ignored prefixes for module index sorting.
-# modindex_common_prefix = []
-
-# If true, keep warnings as "system message" paragraphs in the built documents.
-# keep_warnings = False
-
+html_static_path = ["_static"]
 
-# splhinxcontrib.spelling settings
+pygments_style = None  # covered by sphinx_wagtail_theme
 
 spelling_lang = "en_GB"
```

Comment:
**Suggestion:** per Wagtail core team discussions, we are adopting American English for our docs. I think this should apply to Willow too.

ref: https://github.com/wagtail/wagtail/issues/10608

Reply:
Done 86f3cc8